tests/integration/handlers.js

```diff
@@ -3,3 +3,4 @@ export { handler as performance } from './performance/performance.js';
 export { handler as crypto } from './crypto/crypto.js';
 export { handler as timers } from './timers/timers.js';
 export { handler as fetch } from './fetch/fetch.js';
+export { handler as file } from './file/file.js';
```

Comment:
Should this file be called something like `bufreader.js` instead, given that it doesn't really contain any file-related tests? Or are you planning on adding those in an upcoming PR? If so, seems okay to already be using this name.

Reply:
Good point, maybe `blob` is a better name? This tests buffer reading yes, but indirectly through `Blob` streaming interface - checks if multiple blob streams are handled concurrently without interfering with each other.

Reply:
`blob` would make sense to me, yes. I can land this PR once you've applied that change.